packages/dx-react-chart/src/plugins/stack.tsx

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
 import { Plugin, Getter, Getters } from '@devexpress/dx-react-core';
-import { getStackedSeries } from '@devexpress/dx-chart-core';
+import { getStackedSeries, getStackedDomains } from '@devexpress/dx-chart-core';
 import { StackProps, StacksOptions, OffsetFn, OrderFn } from '../types';
 import {
   stackOrderNone,
   stackOffsetDiverging,
 } from 'd3-shape';
 
+const doGetStackedDomains = ({ domains, series }: Getters) => getStackedDomains(domains, series);
```

Comment:
I think this method name is strange. Can we use another one? For example, 'getDomains'?

Reply:
Renamed.